### PR TITLE
Fix darkmode in Brave Search & Searx

### PR DESCRIPTION
**Pull request by Sevichecc**
- Fixed the automatic dark mode in Brave Search & Searx. 
- Fixed The dark mode CSS selector is not working properly in Brave Search.
- Remove useless text.

### CODE REVIEW

**Comment by Fivefold** [comment]
Thank you @Sevichecc I really appreciate you working on this and taking the trouble of porting it from your miniflux-fork to my repo. Since you seem to be actively working on it, can you @ me here when the PR is ready for review?

**Comment by Sevichecc** [comment]
@Fivefold, This PR seems ready for review. Would you mind taking a look?

And I really like the linkding-injector project. Having little experience developing browser extensions, I’ve learned a great deal from your work. Thank you!

**Review by Fivefold — COMMENTED** [review]
Unfortunately automatic theme detection still doesn't work fully in Brave and SearXNG. When having a dark system theme and having everything on auto the injection box is light (at least for me on windows 10 and Firefox, although I suspect the same happens with chrome. At least it did before this PR).

A table illustrating the states would look something like this (horizontal is Search engine theme, vertical is extension theme)

|       | System | Light | Dark |
|-------|--------|-------|------|
| Auto  | ❌      | ✅     | ✅    |
| Light | ✅      | ✅     | ✅    |
| Dark  | ✅      | ✅     | ✅    |

Steps to reproduce:

1. Set your OS System theme to dark.
2. Set the theme in the search engine to "System" (with Brave) or "Auto" (with SearX)
3. Set the relevant themes in the extension settings to "auto"
4. Check theme of injection box, it should be light

Can you check if you can reproduce this with a dark system theme?

**Comment by Sevichecc** [comment]
That's strange. Everything works fine on my computer and browsers. Perhaps there's an issue with the Windows dark mode?

My OS and browser versions are:
• OS: macOS Ventura 13.5
• Brave Browser: v1.52.117 
• Firefox: v112.0.1 (64-bit)
• Chrome: v114.0.5735.90

The color mode automatically switches between dark and light mode as follows:

https://github.com/Fivefold/linkding-injector/assets/91365763/d275a066-5134-4d92-a659-92353887a984


https://github.com/Fivefold/linkding-injector/assets/91365763/02286713-e4e7-4b85-ae7f-6b3ce2fb0c8b

File: src/options.svelte
Comment (Fivefold) [inline src/options.svelte:1]: Unless you have access to a Windows machine and are willing to troubleshoot this, I'd propose we revert the notice here (even though it doesn't apply to macOS).

**Review by Fivefold — COMMENTED** [review]
Ah, the joys of cross-OS differences. See my other comment regarding options.svelte.

Otherwise I'm happy to merge this. Thank you again for your time and work

**Pull request merged**